CHANGELOG.md

```diff
@@ -18,6 +18,7 @@ Changes
 * [UI]: Sorted columns for advanced charts and added labels to tiny charts on admin statistics page.
 * [UI]: Fixed: Allow organism name not included in the current taxonomy.
 * [Developer]: Removed `prop-types` as a front-end development dependency.
+* [Developer]: Updated nodejs to `v16.14.2` in the `pom.xml` file.
```

Comment:
why to this version, the current LTS version is 16.15.0?  Did that just get updated?

Reply:
Looks like `16.15.0` just got released yesterday https://github.com/nodejs/node/releases. I can update to that one.

Reply:
Please.